src/e2e_test/adaptive_export_loadtest/README.md

```
@@ -0,0 +1,72 @@
+bash harness/nfr.sh
+# 4. no-loss — deterministic PEM↔ClickHouse row-level reconciliation for the DX arm
+bash harness/exp_row_reconcile.sh
+```
+
+**Scripts:** `log4shell_fire.sh` (attack-signal generator, bob#140-hardened) · `exp_matrix.sh` (reduction matrix, the canonical ALL-vs-DX runner) · `nfr.sh` (throughput/mem/latency) · `exp_row_reconcile.sh` (no-loss).
+
+> The DX arm needs the load-gen pods bound to a **benign User SBoB** (`kubescape.io/managed-by: User`, `rulePolicies.R0002.processAllowed`) or benign noise gets steered and contaminates the reduction — see `biz/PoC/log4j/datavolume/denoise_sbobs/`.
+
+---
+
+## Layout
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)